rootfs/usr/share/www/observer.css

```diff
@@ -28,8 +28,14 @@ body {
 .disconnected {
     color: red;
 }
+.reason {
```

Comment:
Override the link color explicitly.

```suggestion
.reason a {
```

Reply:
That excludes the dot